cmd/thanos/bucket.go

```diff
@@ -291,6 +299,123 @@ func registerBucketInspect(m map[string]setupFunc, root *kingpin.CmdClause, name
 	}
 }
 
+// RegisterBucketWeb exposes a web interface for the state of Thanos remote store just like `pprof web`
+//
+// TODOS:
+//
+// 1. Some download code is duplicated from `ls -o json`; refactor as needed
+// 2. Some web code is duplicated from query UI, refactor as needed
+// 3. Do we need any of the route prefix?
+func registerBucketWeb(m map[string]setupFunc, root *kingpin.CmdClause, name string, objStoreConfig *pathOrContent) {
+	cmd := root.Command("web", "Web interface for remote storage bucket")
+	bind := cmd.Flag("listen", "HTTP host:port to listen on").Default("0.0.0.0:8080").String()
+	interval := cmd.Flag("refresh", "Refresh interval in minutes").Default("30").Int()
```

Comment:
Why not `.Duration` so user can choose unit themselves?

Reply:
I wasn't aware of a `.Duration()`, that does make things easier 👍